doc/lua_api.txt

```diff
@@ -9299,20 +9299,31 @@ Chat command definition
 
 Used by `minetest.register_chatcommand`.
 
+Specifies the function to be called and the privileges required when a player
+issues the command.  A help message that is the concatenation of the params and
+description fields is shown when the "/help" chatcommand is issued with the
+command name as its argument, 
+
     {
-        params = "<name> <privilege>",  -- Short parameter description
+        params = "",
+		-- Short parameter description.  See the below note.
 
-        description = "Remove privilege from player",  -- Full description
+        description = "",
+		-- Full description of the command's purpose.
```

Comment:
```suggestion
		-- Textual description of the command's purpose.
```
short <-> full suggests a relation to "params" which isn't there

Reply:
Since I already added "purpose", that should set the two apart.  Would you agree to "General description"?